ReactiveCocoa/Swift/Property.swift

```diff
@@ -241,12 +241,15 @@ public func <~ <P: MutablePropertyType>(property: P, producer: SignalProducer<P.
 	let disposable = CompositeDisposable()
 
 	producer.startWithSignal { signal, signalDisposable in
-		property <~ signal
+		disposable += property <~ signal
```

Comment:
Maybe do:
```swift
producer
  .on(completed: { disposable.dispose() })
  .startWithSignal { ... }
```

That way you avoid the extra call to `observeCompleted` and needing to add that to the disposable?